packages/admin/src/common/toolbar/backbutton/ToolbarBackButton.styles.tsx

```diff
@@ -0,0 +1,45 @@
+import { Theme } from "@material-ui/core/styles";
+import { ArrowBack } from "@material-ui/icons";
+import { makeStyles } from "@material-ui/styles";
+import * as React from "react";
+
+import { useComponentThemeProps } from "../../../mui/useComponentThemeProps";
```

Comment:
Isn't there are `@src/` path alias in comet-admin 🤔

Reply:
there currently is no `@src/`